### PR TITLE
Add OVO as our user

### PR DESCRIPTION
**Pull request by sghintala**

### CODE REVIEW

**Comment by sghintala** [comment]
@hsluoyz  can you help with below error, which I got in above build : 

`remote: Invalid username or password.
fatal: Authentication failed for 'https://hsluoyz@github.com/casbin/casbin.github.io.git/'`

File: website/data/users.js [hunk 1]
```diff
@@ -77,4 +77,10 @@ module.exports = [
     infoLink: 'https://github.com/casbin/casbin/issues/302',
     pinned: true,
   },
+  {
+    caption: 'OVO',
+    image: '/img/users/ovo.png',
+    infoLink: 'https://ovo.id',
```
Comment (hsluoyz) [inline website/data/users.js]: The `infoLink` should be a page that shows ovo uses Casbin. See others.

Comment (sghintala) [reply website/data/users.js]: @hsluoyz I can not share a page that show's OVO is using casbin, because it is a private repo.

Comment (hsluoyz) [reply website/data/users.js]: ok

**Comment by hsluoyz** [comment]
Merged.

**Pull request merged**